test-suite/unit-tests/Makefile

```diff
@@ -0,0 +1,42 @@
+# Makefile for unit tests
```

Comment:
There are no makefiles in sub-directory of the test-suite. Everything is in the main Makefile in the `test-suite/` directory.

Reply:
OK.